standup/status/jinja2/users/login.html

```diff
@@ -0,0 +1,43 @@
+{% extends "base.html" %}
+
+{% block title %}Sign in | {{ super() }}{% endblock %}
+
+{% block more_styles %}{% stylesheet 'browserid' %}{% endblock %}
+{% block before_content %}{% endblock %}
+{% block content %}
+  <div class="grid_12"><h1>Sign in</h1></div>
+
+  <div class="grid_6">
+    <div class="box content-box">
+      <h2>Persona</h2>
+      <p>You can login using <a href="https://login.persona.org/">Mozilla Persona</a>. This service will be going away later this year however.</p>
+      <p>
+        If you've logged into the site before, you should use this method now. You'll be prompted after
+        signing in to associate a Github account. Doing this is the best way to ensure that your existing
+        account is associated with your Github account.
```

Comment:
"... with your Github account and you will be able to continue using your account after Persona dies a raging fiery death of glory."